kernel/poll.c

```diff
@@ -172,11 +172,11 @@ static inline void clear_event_registration(struct k_poll_event *event)
 
 /* must be called with interrupts locked */
 static inline void clear_event_registrations(struct k_poll_event *events,
-					      int last_registered,
-					      k_spinlock_key_t key)
+					     int num_events,
+					     k_spinlock_key_t key)
 {
```

Comment:
None of these changes have any significance, so they really should be in a separate commit or left out of the PR.

Reply:
Fixed.